qemu/tests/qemu_guest_agent.py

```diff
@@ -721,15 +698,35 @@ def _action_after_fsfreeze(self, *args):
             raise error.TestFail("FS freeze failed, guest still can"
                                  " write file")
 
-    @error.context_aware
+    def gagent_check_reset_reboot_shutdown(self, test, params, env):
+
+        """
+        Execute "system_reset" and send "shutdown,reboot" command to guest agent after FS freezed
+        :param test: kvm test object
+        :param params: Dictionary with the test parameters
+        :param env: Dictionary with test environment.
+        """
+        vm = env.get_vm(params["main_vm"])
+        if not vm.is_alive():
+            test.skip("VM died,can't continue the test loop.")
+        self.gagent.fsfreeze()
+        self.gagent.verify_fsfreeze_status(self.gagent.FSFREEZE_STATUS_FROZEN)
```

Comment:
you are right,  fsfreeze() function verified guest file system status after fsfreeze command.  I haven't to verify it again.   deleted this line.